mypyc/test-data/irbuild-bytes.test

```diff
@@ -33,6 +33,7 @@ L0:
     b5 = r9
     return 1
 
+
```

Comment:
Style nit: we tend to use a single empty line between irbuild tests.

Reply:
Done!